hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestMORDataSource.scala

```diff
@@ -562,4 +564,25 @@ class TestMORDataSource extends HoodieClientTestBase {
     df.show(1)
     df.select("_hoodie_commit_seqno", "fare.amount", "fare.currency", "tip_history").show(1)
   }
+
+  @Test def testQueryMorWithoutStar(): Unit = {
```

Comment:
- Rename to `testQueryMORWithBasePathAndFileIndex`
- Can we make this parameterized and more elaborate like the test you have written in COW testsuite ?